examples/platform/silabs/display/lcd.h

```diff
@@ -65,6 +65,7 @@ class SilabsLCD
     int DrawPixel(void * pContext, int32_t x, int32_t y);
     int Update(void);
     void WriteDemoUI(bool state);
+    void WriteDemoUI();
```

Comment:
Right above there is `Update(void)`, we might want to define our coding standard regarding these declarations.

Reply:
I agree. In my opinion, these class / folder would need a clean and documentation. It is going to in a follow up to avoid muddying the waters of this PR.